packages/controllers/organization_detail.coffee

```diff
@@ -5,7 +5,60 @@ if Meteor.isClient
   Template.organizationDetail.helpers
     organization: ->
       Organizations.findOne(@organizationId)
+    userIsMember: ->
+      UserProfiles.findOne({
+        userId: Meteor.userId(),
+        memberOfOrgs: @organizationId
+      })
+    userIsAdmin: ->
+      UserProfiles.findOne({
+        userId: Meteor.userId()},
+        $in: {adminOfOrgs: [@organizationId]}
+      )
```

Comment:
also, what would you think of having this in the model? it might need to have the user id as an argument, but I think it could work.